Assets/Scripts/Climbing.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Climbing : MonoBehaviour {
+    [SerializeField]
+    private float ClimbingForce = 5;
+    [SerializeField]
+    private Rigidbody2D rigidbody2DInstance;
+    [SerializeField]
+    private float maxClimbSpeed = 5;
+    [SerializeField]
+     private Animator anim;
+
+
+   
+    private float verticalInput;
+    private bool isOnLadder;
+    // Use this for initialization
+
+   
+    // Update is called once per frame
+    void Update () {
+        UpdateVerticalInput();
+       
```

Comment:
be consistent with where you have brackets. sometimes you have after the function and sometimes you put it below the function